CHANGELOG.md

```diff
@@ -63,9 +72,9 @@ Thanks also to @DABH, @wbt, and @fearphage for contributions and reviews!
 
 ## v3.3.4 / 2022-01-09
 
-Compared to v3.3.3, this version fixes some issues and includes some updates to project infrastructure, 
-such as replacing Travis with Github CI and dependabot configuration. 
-There have also been several relatively minor improvements to documentation, and incorporation of some updated dependencies.  
+Compared to v3.3.3, this version fixes some issues and includes some updates to project infrastructure,
+such as replacing Travis with Github CI and dependabot configuration.
+There have also been several relatively minor improvements to documentation, and incorporation of some updated dependencies.
```

Comment:
FYI, you should turn off automatic trimming of trailing whitespace in .md files in your IDE, as those are significant in the MD format.